warrior/Framework/Utils/cli_Utils.py

```diff
@@ -662,25 +663,75 @@ def _send_cmd(obj_session, **kwargs):
 
 def _get_response_dict(details_dict, index, response, response_dict):
     """Get the response dict for a command. """
+    def print_warn_msg(keyvars, numpats):
+        """ print a warning message if the number of vars to be stored with
+        the patterns does not match with the number of patterns
+        """
+        warn_msg = ("The number of variables to store is {0} than the patterns.\n"
+                    "The number of keys({1}) is {2} which is {0} than number of patterns {3}")
```

Comment:
@navvrnd This could be rephrased with meaningful contents like - 'response reference variable' & 'response reference keys'

Reply:
done.